RestSharp/Serialization/Xml/DotNetXmlSerializerClientExtensions.cs

```diff
@@ -20,7 +20,7 @@ public static class DotNetXmlSerializerClientExtensions
                 .WithXmlSerializer(xmlSerializer)
                 .WithXmlDeserializer(xmlDeserializer);
 
-            return restClient.UseSerializer(serializer);
+            return restClient.UseSerializer(() => serializer);
```

Comment:
Why do we need a factory? What is the advantage to instantiate it all over again? Serializers are usually singletons.

Reply:
Haha, silly me. I made it a factory. Still, would be nice to get a reply from you.